ambry-router/src/main/java/com/github/ambry/router/SimpleOperationTracker.java

```diff
@@ -576,10 +476,12 @@ public void onResponse(ReplicaId replicaId, TrackedRequestFinalState trackedRequ
         if (trackedRequestFinalState == TrackedRequestFinalState.NOT_FOUND) {
           totalNotFoundCount++;
           if (replicaId.getDataNodeId().getDatacenterName().equals(originatingDcName)) {
-            originatingDcNotFoundCount++;
+            if (originatingDcLeaderOrStandbyReplicas.contains(replicaId)) {
+              // Since bootstrap replicas could still be catching up, treat NotFound responses only from leader or
+              // standby replicas as valid.
+              originatingDcNotFoundCount++;
```

Comment:
Thanks @justinlin-linkedin for pointing offline that we should increment not found only if they are from Originating DC.